CONTRIBUTORS.md

```diff
@@ -1 +1,2 @@
 ## Contributors
+1. Aneesh02
```

Comment:
You can remove that extra `1.` from the beginning of your username
Everything else is good. We can merge your PR :smile:

Reply:
Well, there have been some changes to the format.
I have resolved this merge conflict for you :+1:

Reply:
Thanks